### DISCUSSION TITLE
QuML specification changes proposed

### DISCUSSION THREAD

**Discussion by sajeshkayyath** [sunbird-specs/QuML#8] [General]
@vrayulu / @surendrasinghs 

There are a few differences that we identified when we compared the QuML specification against the current inQuiry implementation. Based on the analysis we have come up with a proposal for a spec revision as captured below.

https://project-sunbird.atlassian.net/wiki/spaces/QB/pages/3299803150/Inquiry+-+QuML+Compliance

Looking forward for a review and feedback on the same. 

cc @krgauraw / @pallakartheekreddy / @PoojaJManjooran / @kameshbhr / @rajnishdargan / @vaibhavbhuva

**Comment by krgauraw** [comment]
@vrayulu @surendrasinghs 
Based on reviews, the below document updated for spec changes
https://project-sunbird.atlassian.net/wiki/spaces/QB/pages/3299803150/Proposed+QuML+Spec+Changes

Requesting you to review once and create v1.1 branch.
cc:  @pallakartheekreddy / @PoojaJManjooran / @kameshbhr / @rajnishdargan / @vaibhavbhuva / @sajeshkayyath

**Reply by vrayulu** [reply]
Thanks @krgauraw . created the v1.1 branch for QuML specs (https://github.com/sunbird-specs/QuML/tree/v1.1). Please send the spec updates as PRs to this branch.

**Reply by vrayulu** [reply]
@krgauraw - also, please create an issue in https://github.com/sunbird-specs/QuML/issues (before sending the PR) listing the high level changes being proposed to the spec.

**Reply by krgauraw** [reply]
Hi @vrayulu Please find below PR. Issue is also created (https://github.com/sunbird-specs/QuML/issues/9)
PR: https://github.com/sunbird-specs/QuML/pull/10

cc: @pallakartheekreddy / @PoojaJManjooran / @kameshbhr / @rajnishdargan / @vaibhavbhuva / @sajeshkayyath
